.github/workflows/doc.yml

```diff
@@ -0,0 +1,41 @@
+name: Docs
+on:
+  push:
+    paths:
+      - 'doc/'
```

Comment:
I think it would be useful to add a workflow file `.github/workflows/doc.yml` to check the operation of CI.

Reply:
```suggestion
      - 'doc/'
      - '.github/workflows/doc.yml'
```